src/Evolution/DiscontinuousGalerkin/Limiters/WenoHelpers.hpp

```diff
@@ -55,7 +55,8 @@ void reconstruct_from_weighted_sum(
         std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>,
         Variables<TagsList>,
         boost::hash<std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>>>&
-        neighbor_vars) noexcept {
+        neighbor_vars,
+    const DerivativeWeight derivative_weight) noexcept {
```

Comment:
Clang-format has aligned this arg consistently with the previous args. Those fail to show up in the diff because the preceding arg `neighbor_vars` takes up so many lines...

Can you clarify what looks off to you?

Reply:
Ah, I see. I misread things, sorry :)